changelog.d/mod/2990.enhance.rst.txt.txt

```diff
@@ -0,0 +1 @@
+Added a command to toggle kickbans sending an embed to the user detailing the reason for the kickban, the server they were kickedbanned and if it was a kick or a ban.
```

Comment:
Just a nitpick but you should be explicity about the full qualified name here, instead of just saying `a command`

Reply:
btw, this file has wrong extension